changelogs/changelog_2_0_0_details.md

```diff
@@ -344,6 +344,7 @@
   + Added `toDateString`, `toISOString`, `toJSON`, `toTimeString`, `toUTCString` converters for `DateTime`.
 - Added `BackwardsIndex` overload for `CacheSeq`.
 - Added support for nested `with` blocks in `std/with`.
+- Added `ensureMove` to the system module. It ensures that the parameter is moved, otherwise an error is given at the compile time.
```

Comment:
```suggestion
- Added `ensureMove` to the system module. It ensures that the passed argument is moved, otherwise an error is given at the compile time.
```